8.8/apache-buster/Dockerfile

```diff
@@ -57,17 +56,23 @@ RUN { \
 		echo 'opcache.fast_shutdown=1'; \
 	} > /usr/local/etc/php/conf.d/opcache-recommended.ini
 
-WORKDIR /var/www/html
-
 # https://www.drupal.org/node/3060/release
 ENV DRUPAL_VERSION 8.8.8
-ENV DRUPAL_MD5 c8265f0228ca0306de7b2755e4731a3d
 
+# https://github.com/drupal/drupal/blob/9.0.1/composer.lock#L4052-L4053
+COPY --from=composer:1.10 /usr/bin/composer /usr/local/bin/
 RUN set -eux; \
-	curl -fSL "https://ftp.drupal.org/files/projects/drupal-${DRUPAL_VERSION}.tar.gz" -o drupal.tar.gz; \
-	echo "${DRUPAL_MD5} *drupal.tar.gz" | md5sum -c -; \
-	tar -xz --strip-components=1 -f drupal.tar.gz; \
-	rm drupal.tar.gz; \
-	chown -R www-data:www-data sites modules themes
+	export COMPOSER_HOME="$(mktemp -d)"; \
+	cd /var/www/; \
+	# composer won't install to a non-empty dir: 'Project directory "/var/www/" is not empty.'
+	rmdir /var/www/html; \
+	composer create-project --no-install "drupal/recommended-project:$DRUPAL_VERSION" ./; \
```

Comment:
Hopefully the time period between releases to https://github.com/drupal/drupal/releases (and I guess more relevantly, new version numbers presented in the update API we're scraping) and new tags on https://github.com/drupal/recommended-project/releases is short. :sweat_smile:

Reply:
> This repository is auto-generated based on https://git.drupalcode.org/project/drupal/-/tree/8.8.x/composer%2FTemplate%2FRecommendedProject

So, I would hope this will be updated in a timely fashion on each release.